pvt/helpers/src/models/types/TypesConverter.ts

```diff
@@ -20,6 +20,11 @@ import {
 } from '../tokens/types';
 import { ZERO_ADDRESS } from '../../constants';
 
+export function computeDecimalsFromIndex(i: number): number {
+  // Produces sequential series (18,17,16,...,0,17,16,15,...,1,18,17,16,15,...)
```

Comment:
I'm confused by this comment... the line below produces the series 18,17,16,...,3,2,1 over and over, right? But the comment shows it going to zero, then starting at 17 the next time, then going to 1 and restarting at 18.

Reply:
Oh, sorry for the confusion. Your line actually does produce 18..0, followed by 17..1, and thereafter 18..1 repeatedly. Is that actually the desired behavior? If so, my reduction below is incorrect. But why is that desirable?

Reply:
Perhaps you want the infinitely repeating series 18..0?

```
return 18 - (i % 19);
```

Reply:
Yes; that's much cleaner. This was one of those late night things I just didn't spend time on making better (partly because the exact sequence really doesn't matter; just should contain all possible decimal values). Originally it was developed for the 20-token pool, so it went 18-0 and stayed at 0. I didn't want that pattern for the 100-token pools (though I guess it would have been fine).

Anyway that's much more readable, so I'll change it.